util/deephash/deephash_test.go

```diff
@@ -552,7 +552,7 @@ func TestGetTypeHasher(t *testing.T) {
 			out:  "\x01\x00\x00\x00\x02\x00\x00\x00\x03\x04\x00\x00\x00",
 		},
 		{
-			name: "IntIntByteInt-canddr",
+			name: "IntIntByteInt-canaddr",
```

Comment:
This seemed to match something below, but maybe it's intentional. If it is, a comment explaining it would be helpful...

Reply:
Canaddr is correct